src/Api/ErrorListener.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * @Author: Erik Slovak <erik.slovak@technologystudio.sk>
+ * @Date: 2024-11-29T20:43:00+01:00
+ * @Copyright: Technology Studio
+**/
+
+import {
+  ServiceOperationError,
+  containsNonNetworkError,
+} from '@txo/service-prop'
+import { subscribeErrorListener } from '@txo-peer-dep/error-handler'
+import * as Sentry from '@sentry/react-native'
+import { VALIDATION_ERROR } from '@txo/service-graphql-react'
```

Comment:
probably better to move into @txo-peer-dep/error-handler so we don't have dependency on this package.